src/collapse-perf.rs

```diff
@@ -18,7 +17,7 @@ const TIDY_JAVA: bool = true;
         perf script -F comm,pid,tid,cpu,time,event,ip,sym,dso,trace
     If you save this output add --header on Linux >= 3.14 to include perf info."
 )]
-struct Opt {
+pub struct Opt {
```

Comment:
Why does this need to be `pub`?

Reply:
It doesn't. I think I initially moved it to lib.rs and mad it public, but forgot to remove pub when I moved it back.